### PR TITLE
Feature/blaze docs

### PR DESCRIPTION
**Pull request by hhuuggoo**

### CODE REVIEW

File: sphinx/source/docs/user_guide/server.rst [hunk 1]
```diff
@@ -101,8 +101,63 @@ Streaming Data
...
+Blaze Server
+------------
+
+To use the smart downsampling facilities of bokeh, you must install the multi-user blaze server, https://github.com/ContinuumIO/multiuserblazeserver.::
+
+    $ conda install -c bokeh multiuserblazeserver
+
+or::
+
+    $ pip install -c bokeh multiuserblazeserver
```
Comment (damianavila) [inline sphinx/source/docs/user_guide/server.rst]: pip does not have channels, AFAIK, this should be: `pip install multiuserblazeserver`

**Comment by damianavila** [comment]
Btw, you renamed the examples, you need to skipped from testing, I guess this is why TravisCI is failing...

File: sphinx/source/docs/user_guide/server.rst (hunk 1 above)
Comment (bryevdv) [inline sphinx/source/docs/user_guide/server.rst]: we really, really have to rename this package "`multiuserblazeserver`" is reasonable.

Comment (bryevdv) [inline sphinx/source/docs/user_guide/server.rst]: We should coordinate with mrocklin so there is just `blaze-server` with a multi-user mode

**Comment by hhuuggoo** [comment]
@damianavila can we merge this?

**Comment by bryevdv** [comment]
Can we rename `multiuserblazeserver` to something reasonable before we start publishing docs about it.

**Comment by hhuuggoo** [comment]
@bryevdv we'll have to sort that out with @mrocklin

**Comment by bryevdv** [comment]
@hhuuggoo The plan is still to move this functionality directly into the `bokeh` repo (but distribute a separate package), correct? Should we close this PR, or mark it WIP?

**Comment by hhuuggoo** [comment]
can we keep it open - I should have something next week for the new changes

**Comment by bryevdv** [comment]
Will incorporate this info into current ongoing user guide rewrite (and supersede this PR)

**Comment by damianavila** [comment]
in #2257?

**Pull request closed without merging**